### PR TITLE
Added my name and repo link to Reefit Light/Dark mode

### PR DESCRIPTION
**Pull request by Terieyenike**
This submission is for the light/dark mode feature.

### CODE REVIEW

**Comment by MattCSmith** [comment]
@Terieyenike I think you entry should be on line 31, underneath the light/dark mode heading 🙂

File: README.md [hunk 1]
```diff
@@ -7,105 +7,108 @@ Student submissions for the JavaScript 20 projects
...
 
 
-### Joke Teller - Original: [Github](https://github.com/JacintoDesign/joke-teller/)  
+### Joke Teller - Original: [Github](https://github.com/JacintoDesign/joke-teller/)
 
 *
 
 
-### Light/Dark Mode - Original: [Github](https://github.com/JacintoDesign/light-dark-mode)   
+### Light/Dark Mode - Original: [Github](https://github.com/JacintoDesign/light-dark-mode)
```
Comment (MattCSmith) [inline README.md:31]: I think your entry should be here

File: README.md [hunk 2]
```diff
@@ -7,105 +7,108 @@ Student submissions for the JavaScript 20 projects
...
 
 
-### Splash Page - Original: [Github](https://github.com/JacintoDesign/splash-page)  
+### Splash Page - Original: [Github](https://github.com/JacintoDesign/splash-page)
 
 *
 
 
 
+### Teri Eyenike - Original: [Github](https://github.com/Terieyenike/Reefit)
```
Comment (MattCSmith) [inline README.md:111]: And then remove it from here.

File: README.md (hunk 1 above)
Comment (Terieyenike) [reply README.md:31]: Looking into it.

Thanks

**Pull request merged**